package.json

```diff
@@ -99,8 +87,8 @@
     "prettier": "prettier \"**/*.{json,md,js,yml}\"",
     "preversion": "yarn && yarn test && yarn dist",
     "start": "pm2 start",
-    "start:dev": "cross-env NODE_DEBUG=\"@wireapp/*\" ts-node src/index.ts",
-    "test": "ts-node -P tsconfig.jasmine.json node_modules/jasmine/bin/jasmine --config=jasmine.json",
+    "start:dev": "node dist/main.js",
```

Comment:
Why not using `ts-node` here?

Reply:
And why have you removed the `NODE_DEBUG` setting?

Reply:
it was part of the mass remove.. yeap :)